guide/whatsnew.md

```diff
@@ -0,0 +1,42 @@
+## What's New?
+
+The version of DIDComm incubated in the Hyperledger Aries community is referred to as Version 1 (V1). This spec describes the next version, referred to as Version 2 (V2). This section will describe the changes between V1 and V2, useful to members of the Aries community.
+
+### Summary of Changes
+
+- Formalization of methods used in V1
+  - JWM based envelope
+  - ECDH-1PU standardized form of AuthCrypt
+- Both DID and key in each message
+- Special Handling of Peer DIDs eliminated
+- Message structure split between 'headers' and body.
+- No AnonCrypt encryption method.
+
+### Practical Changes
+
+The list of changes above leads to practical changes in how DIDComm is used.
+
+#### DID Exchange not needed
+
+Each message contains both the sender key (used in the encryption layer), and the sender's DID. The exchange of DIDs that occurs via the DID Exchange Protocol used in V1 occurs in each message that is transferred. The important step of rotating DIDs is accomplished via the `prior_did` header that travels alongside any protocol message. These features make the DID Exchange Protocol redundant.
+
+One side effect of the DID Exchange Protocol in V1 was that you confirmed the validity of the DID with a round trip to the other party. Many protocols will provide this assurance via the flow of the protocol prior to the point where round-trip testing is required. When this round-trip is desired prior to the beginning of a protocol, a round trip with another protocol (such as Trust Ping or Feature Discovery) can provide the same assurance.
+
+#### Process From Headers prior to Protocol Processing
+
+Relationship changes in V1 were handled inside the DID Exchange Protocol. In V2, relationship changes including discovery and rotation are handled in message headers.
+
+In V2, messages must evaluate the `from` and `from_prior` headers of every message prior to beginning the protocol message processing. 
+
+#### No technical difference between Ephemeral Mode and Full Mode
+
+Ephemeral mode in V1 was a method of passing messages without first performing an exchange of DIDs. Given that we no longer have a need to perform an exchange of DIDs prior to passing messages of another protocol, we no longer need to designate a mode for ephemeral interactions.
+
+#### Use Peer DIDs (or other suitable DID method) in place of AnonCrypt
```

Comment:
We still cannot use `did:peer` as I said in https://github.com/decentralized-identity/didcomm-messaging/issues/33#issuecomment-680052846.

The `from` and `from_prior` fields would need to be tweaked such that DID URLs be used instead of just DIDs.

And even then, I'm not so sure of the implications of such a tweak...

Reply:
Well... I think the `initial-state` idea was completely scrapped from did-core... so now we depend on `did:peer` to provide the initial state in a DID.

Reply:
See relevant discussions for `did:peer` in:

* https://github.com/decentralized-identity/peer-did-method-spec/issues/24
* https://github.com/decentralized-identity/peer-did-method-spec/pull/23

Reply:
given wg call discussion on 2020/09/14, the DIDComm JWM profile needs to be adjusted to accommodate whatever method the DID spec/community chooses to pass off/pre ledger recorded information.

Reply:
As we talked about on the WG call, when we change to a DID URL we'll want to specify that during the "from" validation process the DID must be resolved to a full did document rather than de-referenced to only the key included in that DID URL. I agree that we're going to need to carry the parameters through though and changing from DID to DID-URL would allow this to occur.